packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -26,13 +26,14 @@ export async function getWorkspaceOrgType(
   const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
 
   if (isNullOrUndefined(username)) {
-    telemetryService.sendError(
+    telemetryService.sendException(
+      'get_workspace_org_type',
       'workspaceOrgType.getWorkspaceOrgType ran into an undefined username.'
     );
   }
 
   const isScratchOrg = await OrgAuthInfo.isAScratchOrg(username!).catch(err =>
-    telemetryService.sendError(err)
+    telemetryService.sendException('get_workspace_org_type_scratch_org', err)
```

Comment:
Is this sending the error object or the message?

Reply:
Oh right, let me update it to just send a message.